\chapter{Discussion}
\label{chap:discussion}

%---------------------
%
% DISCUSSION
%
%---------------------

In this final chapter of my thesis, I first discuss the implications of the results of Chapters \ref{chap:TEM_paper} and \ref{chap:missing_mammals} and then of Chapter \ref{chap:STD_paper}.
In both sections, I discuss the methodological caveats of my approaches and propose future research avenues to solve these problems and expand into new areas.
Finally, I discuss the overall importance of combining both living and fossil species into macroevolutionary studies.

\section{The future of the Total Evidence method}
The Total Evidence method is a promising way of testing macroevolutionary hypotheses using extinct and extant taxa \citep[e.g.][]{ronquista2012,Slater2012MEE,Wood01032013,beckancient2014,Dembo2015}.
However, as shown in the Chapters \ref{chap:TEM_paper} and \ref{chap:missing_mammals}, this method is sensitive to missing data.
As discussed in both chapters, increasing the number of morphological characters and the number of living taxa with coded morphological characters is the most effective way to recover the correct topology.
This data is available and reasonably easy to access in natural history museum collections worldwide. 
In addition, software infrastructure has been developed to facilitate collaborative data collection of this kind \citep[\textit{MorphoBank};][]{morphobank}.
Therefore, hopefully the scarcity of morphological data for living species will be gradually solved with time, provided the funding exists to make such efforts possible.
However, there are other general problems with Total Evidence phylogenies that were not developed in the discussions of Chapters \ref{chap:TEM_paper} and \ref{chap:missing_mammals}. 

Firstly, dating Total Evidence phylogenies is difficult. 
Because these phylogenies contain both living and fossil taxa, the preferred way to date them is to use the tip-dating method \citep[e.g.][]{ronquista2012,Wood01032013,Dembo2015}.
This method relies on the age of the fossil taxa (treated as tips) to date the divergence times of the nodes, rather than defining node calibrations \textit{a priori} \citep[cf. node-dating;][]{ronquista2012}.
The tip-dating method has two main advantages: (1) it has been observed using empirical data that it improves the ability to recover the correct topology because it can use the stratigraphic age of the fossil taxa to favour some topological solutions more than others, typically by minimising implied ghost lineages \citep[][and personal communications]{BEASTmaster}; and (2) it reduces the confidence intervals of the node ages compared to a classic node-dating approach \citep{ronquista2012}.
This second point, however, has been questioned by \cite{Arcila2015131} who compared the tip-dating method and the latest models for the node-dating method, for example, the fossilised birth-death model \citep{heaththe2013}, and found the opposite effect, i.e. an increase in node age confidence interval with the tip-dating method.
It would therefore be interesting to run a similar analysis to the simulations in Chapter \ref{chap:TEM_paper} but adding a dating aspect to it.
By comparing dated Total-Evidence matrices using both node-dating and tip-dating, one could formally test the two advantages outlined above as well as their resilience to missing data.

Secondly, a more general problem is that the Total Evidence method relies on the M\textit{k} model \citep{lewisa2001} to measure the morphological distances among taxa.
This model is a generalisation of the Jukes-Cantor model \citep[JC69;][]{jukes1969evolution} that allows a single mutation rate $\mu$ among all character states.
The JC69 model is a great simplification of reality and has been replaced in molecular phylogenetics with more complex models that are closer to biological reality \citep[e.g. the GTR model that allows a different rate for each different type of nucleotide change;][]{tavare1986}.
It is therefore likely that the M\textit{k} model is also a crude underestimation of the complex reality of morphological evolution, especially as the assumption that there is a unique transition rate between character states has been shown to be wrong in at least some cases \citep[e.g. for Dollo traits that are irreversible;][]{WrightDollo}.
\cite{spencerefficacy2013} even demonstrated that non-probabilistic methods such as maximum parsimony outperform the M\textit{k} model for the placement of contentious fossils such as \textit{Archaeopteryx}.
However, more recent and thorough simulations have demonstrated the opposite and have shown that using the M\textit{k} model consistently outperforms maximum parsimony, especially in the presence of missing data \citep{wrightbayesian2014}.
Additionally, incremental improvements on the M\textit{k} model are currently being made to solve at least the fact that the some character states are irreversible once evolved \citep[][and personal communications]{Klopfstein12082015}.

Despite the three major caveats discussed above (missing data, dating methods, and models for morphological evolution), the Total Evidence method remains one of the best methods for including the diversity of life both past and present into phylogenies.
Even though this method was first proposed decades ago \citep[e.g.][]{eernissetaxonomic1993}, recent successful software implementations \citep{Ronquist2012mrbayes,BEAST2} have allowed it to be more widely used in the last three years \citep{ronquista2012,Slater2012MEE,Wood01032013,schragocombining2013,beckancient2014,Arcila2015131,Dembo2015,Klopfstein12082015,Carrizo,Wittenberg2015TEM,gavryushkina2015bayesian}.
This increasing number of studies using the Total Evidence method will probably result in further improvements and popularity as time goes on.
This is really encouraging because it will eventually result in more accurate phylogenies (based on molecular and morphological data), including both living and fossil species, being available for macroevolutionary studies in the near future. 


\section{The cladisto-space as a proxy for describing macroevolutionary changes} 
One important point to keep in mind when building phylogenies, however, is that phylogenetic trees (whether they use all the available data or not) are tools for observing evolutionary patterns and testing evolutionary hypotheses, not the end point of the scientific endeavour.
Until now, only few studies have used a Total Evidence phylogeny as a tool for specifically testing macroevolutionary hypotheses \citep[e.g.][]{Slater2012MEE,Wood01032013,Dembo2015}.
In Chapter \ref{chap:STD_paper}, I used two tip-dated Total Evidence trees for testing whether the K-Pg mass extinction event influenced the diversity of mammals.
Rather than using taxonomic richness as my proxy for diversity, I used disparity, i.e. morphological diversity (or in this case cladistic diversity), and tested whether mammalian disparity increased after the K-Pg extinction.
Although it is becoming increasingly common to estimate the disparity of clades in palaeobiology \citep[e.g.][]{Butler2012,brusattedinosaur2012,toljagictriassic-jurassic2013,brusattegradual2014,bensonfaunal2014,Claddis,Close2015}, methods for estimating disparity suffer from several biases that I did not fully discuss in Chapter \ref{chap:STD_paper}.

%Cladisto vs morpho space (kind of)
Describing the shape or form of an organism is not straightforward and many mathematical methods exist for doing it (e.g. Elliptic Fourier; \citealt{Fourier1982}; Procrustes; \citealt{JamesRohlf1993129}; Convex Hull; \citealt{ANDREW1979216}).
By form here, I am referring to biological variations in the morphology of organisms (cf. the shape defined as the 2D outlines of an individual).
In biology, one approach is to describe an organism's form by using geometric morphometrics \citep{zelditch2012geometric}.
This involves collecting Cartesian coordinates of a series of discrete spatial points (landmarks), transforming them to remove the effect of size differences via Procrustes transformation, and then ordinating these transformed coordinates into a smaller number of variables, for example by using Principal Components Analysis (PCA).
The organism's form can then be described as a single value that summarises the matrix \citep[e.g. the sum of the ranges of each PCA axis;][]{zelditch2012geometric}.
When using this approach, form is approximated by actual continuous measurements collected from the organisms \citep[e.g.][]{friedmanexplosive2010,hopkinsdecoupling2013,finlay2015morphological}.
In our case, however, we instead used inter-taxon distances based on discrete morphological features to describe form \citep[i.e. the cladistic disparity method; e.g.][]{foote1997evolution,Wills2001,Wesley-Hunt2005}.
This method is sometimes criticised because the morphological features are not randomly collected: cladistic characters are usually collected to resolve relationships among lineages \citep{O'Leary08022013}.
This might distort reality because some authors will emphasise differences in the taxonomic groups of interest \citep{Hopkins24032015}.
Additionally, when including fossil data to such analyses, the available characters are highly dependent on the quality of the fossil record.
This can be biased against certain type of characters \citep[e.g. soft tissues;][]{sansomfossilization2013} or towards geological strata with more fossils \citep[e.g. \textit{Lagerst\"{a}tten};][]{Butler2012}.
Nonetheless, these two biases are overshadowed by the advantages of having a great deal of easily available morphological data \citep[some morphological matrices have more than 1000 characters; e.g.][]{O'Leary08022013,ni2013oldest} that can be compared among many taxa across many taxonomic levels \citep[e.g. across all mammals;][]{O'Leary08022013,Slater2012MEE,beckancient2014}.
Furthermore, empirical studies have shown that the same signal seems to be captured when using either geometric morphometric or cladistic methods to describe disparity \citep{foth2012different,hetherington2015cladistic}. 

%Which metric?
Secondly, disparity is an abstraction of morphological diversity; it is a unique value that describes the cladisto-space, that is in turn based on a multidimensional transformation of the discrete form differences among the taxa in the analysis \citep{Wills1994,foote1997evolution}.
There are many different ways of doing this, though generally people have used the four metrics proposed by \cite{Wills1994} (sum and products of ranges and variances) to calculate disparity.
I discussed the caveats of using such metrics in Chapter \ref{chap:STD_paper} but I did not explore several other, more general, problems.
In macroevolutionary studies, one of the purposes of disparity metrics is to describe biological changes in the cladisto-space: for example, when species go extinct the occupancy of the cladisto-space can decrease, suggesting a loss of biological forms, or stay constant, suggesting random loss of species across cladisto-space.
It is not clear what variations in these metrics really reflect biologically apart from variation in the range or variance of each dimension of the cladisto-space.
These metrics are actually only describing the dimensions of the ordinated matrix, and are not directly describing the relative placement of the tips or nodes in that space, unlike measures such as the distance between taxa and the centroid. 
The latter may be of more interest when answering particular macroevolutionary questions such as the one tested in Chapter \ref{chap:STD_paper}, thus disparity metrics based on the placement of the taxa in the cladisto-space should be developed further.
Even though some attempts have been made for measuring the effectiveness of disparity metrics \citep{Ciampaglio2001} there has been no systematic assessment of the statistical power of each metric for describing multidimensional space occupancy.
Future developments of disparity-through-time studies require a better understanding of the statistical performance of disparity metrics and which metrics are most appropriate in specific empirical situations.
In the future, I plan to assess the power of the different available disparity metrics \citep[e.g.][]{Wills1994,Ciampaglio2004,Hughes20082013,huang2015origins} through simulation studies by testing their performance at assessing various types of changes in ordinated matrices such as the distribution of the taxa in the cladisto-space (being randomly- or evenly-distributed or clustered) or changes in cumulative (i.e. explanatory) variance among the dimensions of the cladisto-space.

%Transition - back to using both living and fossil species
One final crucial aspect of disparity-through-time studies regards the use of both living and fossil taxa in these studies.
As discussed in Chapter \ref{chap:STD_paper}, palaeontological or neontological data suggest different patterns of diversification in mammals, with diversification either occurring just after the K-Pg event when fossil species are used \citep[suggesting an effect of K-Pg;][]{O'Leary08022013} or before when living species are used \citep[rejecting an effect of K-Pg;][]{meredithimpacts2011,dosReis2014}.
Because it is impossible to be certain which scenario is correct, I argue that using all the available data, i.e. both living and fossil taxa \citep{Slater2012MEE,beckancient2014} is the best way to describe the observed patterns with more confidence.
Future disparity studies should include both living and fossil taxa to get a true understanding of patterns in disparity-through-time.

\section{Future directions for combining living and fossil species in macroevolutionary studies}
Using data from both living and fossil species has lead to substantial improvements in macroevolutionary studies (e.g. improving ancestral characters estimations; \citealt{Finarelli2006}; mode of evolution testing; \citealt{Slater2012MEE,pant2014complex}; or disparity-through-time analyses; \citealt{Mitchell2015}).
However, combining living and fossil species in such studies often requires extra work and specific expertise \citep[e.g. the study by][gathered experts in statistics, bioinformatics, phylogenetics and palaeontology]{ronquista2012} and can suffer from more problems than those arising from using living and fossil data separately (e.g. the missing data problem described in Chapters \ref{chap:TEM_paper} and \ref{chap:missing_mammals}).
Additionally, it could be counterproductive to add living species when studying fossil clades that have very few or no living relatives (e.g. Trilobita; \citealt{hopkinsdecoupling2013}; Pterosauria; \citealt{Butler2012}; etc.).
Likewise, when studying clades that underwent a recent explosive radiation \citep[e.g. Cichlidae;][]{Genner01052007}, it may not be sensible to add fossil species.
In both cases, adding fossil or living species would not significantly change the observed macroevolutionary patterns, but would take a lot of extra time, effort and expertise.
Several studies, however, have suggested that it is still important to combine living and fossil species in analyses with predominantly living or fossil taxa.
For example, when studying Ostracoderma (armoured jawless fishes that went extinct during the Devonian; 358 Ma), some morphological features can only be interpreted in the light of living species and it is therefore important to compare them to living vertebrates to understand vertebrate evolution \citep{Janvier2015}.
On the other hand, even recent groups often have recently extinct members that may change the conclusions of our analyses if we choose to ignore them.
For example, when studying Strepsirrhini (lemurs and lorises), it may be important to include subfossil giant lemurs that went extinct only 600 years ago and were two orders of magnitude bigger than living lemurs \citep{goodman2003introduction}, especially when studying body size evolution.

In general, discarding part of your data on an arbitrary basis is not good scientific practice. 
In addition, the effect of adding fossil species to analyses of living species (and vice versa) cannot be known \textit{a priori}.
Therefore I argue that we should always use both living and fossil species in our analyses wherever feasible.
Our knowledge in biology has tremendously advanced since the last half century ranging from the amazing revelation of the few glimpses of the deep past provided by the fossil record to the understanding of the complexity and dynamics of modern ecosystems.
Because the inherent characteristic of the deep past is to be unknown and mysterious, it is therefore crucial to incorporate all of this knowledge into macroevolutionary studies to continue revealing the complexities of biodiversity.

%\bibliography{References}