\chapter{Introduction}
\label{chap:introduction}

%Some quote from GG Simpson
%"Certainly paleontologists have found samples of an extremely small fraction, only, of the earth's extinct species, and even for groups that are most readily preserved and found as fossils they can never expect to find more than a fraction.""
%But I'm not sure, maybe not quote is better.


%---------------------
%
% GENERAL INTRO - MAKE IT SHORT
% 
%---------------------

Today's amazing biodiversity represents only an overwhelmingly small fraction of the organisms that ever existed \citep{novacek1992ext,raup1993extinction}.
Even though it is widely accepted that the processes that shaped the patterns observed nowadays are influenced by evolutionary history \citep{fritzdiversity2013}, most of the scientific endeavour in biology focus solely on living species.
Ignoring this can lead to misinterpretation of macroevolutionary patterns and processes \citep{benton2015}.
For example, nowadays crocodilians constitute a species poor group \citep[25 species;][]{uetz2010original} with a low range of shapes and environments \citep[marine or freshwater;][]{Martin2008}.
Therefore when studying macroevolutionary patterns among all vertebrates, crocodilians will have a rather ``marginal'' effect.
For example \cite{Wiens2015} suggests that terrestriality is a driver of diversification among living vertebrates, a pattern essentially driven by Aves, Lepidosauria and Mammalia.
However, crocodilians were much more diverse both in terms of species richness \citep[244 species reported in][]{Bronzati2015} or in terms shapes and environments \citep{stubbs2013}.
In the case of \cite{Wiens2015}, not including fossil species, conceal the true history of this clade, and thus, potentially biases the conclusions of the study.

Besides, including fossil species not only accounts for groups that were more diverse in the past, it also highly improves our descriptions of macroevolutionary patterns such as the timing of diversification events \citep[e.g. significantly reducing node age confidence intervals;][]{ronquista2012}, the relationships among lineages \citep[e.g. solving some controversial fossil placement;][]{Dembo2015} or even gives a potential solution for understanding niche occupancy through time \citep[e.g.][]{pearmanniche2008}.
All this studies have led to a recent consensus among scientists that we need to combine both living and fossil species in macroevolutionary analysis \citep{jacksonwhat2006,quentaldiversity2010,dietlconservation2011,slaterunifying2013,fritzdiversity2013,benton2015}.
Yet, in practice, only few studies have actively focused on combining them since the last three years \citep[e.g.][]{ronquista2012,slaterphylogenetic2013,Wood01032013,beckancient2014,Arcila2015131,Dembo2015}. %TG: or not sure if this is the good string of cites. These papers are mainly about TEM and tip-dating.

% TG: add croc figure? with/without fossils?

%\section{Combining data, methods and disciplines}
This scarcity %of practical applications of combining living and fossils species
is probably due to the fundamental differences between the two approaches to study macroevolution: using either living (neontological) or fossil (palaeontological) data.

The paleontological approach% was heavily popularised by \cite{simpson1945} and
 is based on cladistic data of the fossil record (i.e. discrete morphological observation).
It relies on optimal criteria such as maximum parsimony \citep{Hennig1966,felsenstein2004} to resolve the relations among lineages and on stratigraphy to time such trees \citep{GoloboffTNT}.
This approach allows a direct interpretation of macroevolution in deep time and benefits from recent improvements both on data collection \citep[e.g. 4541 characters in][introducing the term ``phenomics'']{O'Leary08022013} and on dating method \citep[e.g. the \textit{cal3} method from][]{Bapst2014}.
However, this approach does rarely takes into account full living diversity \citep[e.g. only 38 out of 351 living primates for 119 fossil in][]{ni2013oldest} and methods suffer from several biases \citep[e.g. parsimony;][]{wrightbayesian2014}.

Conversely, the neontological approach uses the vast amount of available molecular from living species and is based on probabilistic methods (e.g. Maximum Likelihood or Bayesian).
This approach is based on evolutionary models that rely on the differences in DNA to resolve the relations among lineages and on some specific fossils' occurrence dates for timing the lineages divergence \citep[i.e. the molecular clock;][]{zuckerkandl1965}.
There has been enormous improvements of this approach in the last decade on both the evolutionary models \citep[e.g.][]{bapsta2013,stadlerdating2013,heaththe2013} and on which fossils to use to calibrate the trees \citep{Donoghue2007424,Parham01032012}.
However, this approach uses only the ages of certain fossils instead of the vast amount of informations available from the fossil record (e.g. species richness, traits, biogeography, etc).


\section{Phylogenies with living and fossil taxa}
Nonetheless, the last three years have seen the development of the newly trending Total Evidence method \citep{ronquista2012,slaterphylogenetic2013,Wood01032013,schragocombining2013,beckancient2014,Arcila2015131,Dembo2015}.
This methods allows to combine both molecular data from living taxa and morphological data from living and fossil taxa in the same phylogenetic matrices.
It was first developed in the nineties \citep{eernissetaxonomic1993} but only recently successfully implemented in phylogenetic softwares \citep{Ronquist2012mrbayes,BEAST2}.
By using both available neontological and palaeontological data, this methods allows to better study macroevolutionary patterns and processes.
For example, it allowed great improvements on the estimation of divergence event \citep[e.g.][]{ronquista2012}; evolutionary rates \citep[e.g.][]{beckancient2014}; topology \citep[e.g.][]{Dembo2015}; traits evolution \citep[e.g.][]{slaterphylogenetic2013} or even speciation processes \citep[e.g.][]{Wood01032013}.
There is, however, one drawback to this method: because it needs both molecular data for living taxa and morphological data for living and morphological taxa, it is susceptible to suffer from great amounts of missing data.

\subsection{Effects of missing data on topological inference using a Total Evidence approach}
As a first part of this PhD thesis, in the second chapter, I tackled the problem of missing data in Total Evidence matrices.
I ran long term and thorough simulations to test how the topologies inferred from Total Evidence matrices were sensitive to missing morphological data.
I removed morphological data from Total Evidence matrices via three parameters where data are potentially missing: (1) the number of living taxa with molecular data but no morphological data; (2) the amount of missing data in the fossil record and (3) the number of overall morphological characters in the matrix.
I modified the level of data in the three parameters and in their combination and then inferred the phylogenies using both Maximum Likelihood and Bayesian approach.
Finally, I compared how the missing data parameters and their interactions as well as the phylogenetic inference method influenced the ability of estimating the correct tree topology.
I found that the number of living taxa with both morphological and molecular data are essential to recover accurate topologies.
This study rose the question of how can we improve Total Evidence topologies and especially, how much morphological data are available for living taxa?

\subsection{Morphological data availability in living mammals}
Following this question, in the third chapter of this thesis, I monitored how many morphological data were available in the literature for living mammals.
I downloaded all the recent available morphological matrices and counted the number of living mammals with available morphological data at three different taxonomic levels (species, genus and family) for each mammalian order.
For each order with missing data, I measured if the data weren't biased toward some specific clades in each order using phylogenetic structure methods \citep{webb2002phylogenies}.
I found that a lot of living mammals have no morphological data at the species or the genus level but, that at least most of the available data was randomly distributed.
These results highlight the importance of cladistics and collecting morphological data, even in the age of genomics, especially for combining living and fossil data in the same phylogenies.

\section{Total evidence phylogenies applications}
The two previous chapters only focused on the technical and practical side of combining living and fossil taxa into phylogenies and underlined the importance of a good data overlap between living and fossil taxa.
However, many studies have been able to use the Total Evidence method even with low overlap between living and fossil taxa by using strong topological constraints (\citealt{ronquista2012,schragocombining2013,slaterphylogenetic2013,beckancient2014}; but see \citealt{Arcila2015131,Dembo2015}).
This resulted in Total Evidence phylogenies were the topology are based on strong but valid \textit{a priori} topologies (e.g. based on \citealt{meredithimpacts2011} for \citealt{slaterphylogenetic2013}).
The observable patterns in these phylogenies can then be used by biologists to test some hypothesis relating to macroevolutionary processes.

%Because these phylogenetic trees capture macroevolutionary patterns more accurately (\citealt{ronquista2012,schragocombining2013,slaterphylogenetic2013,beckancient2014,Dembo2015}; but see \citealt{Arcila2015131}), they can be used as a base for testing macroevolutionary processes.
One example of pattern observed in phylogenies can be the shift of ecological dominant species through time due to drastic biotic or abiotic changes in the biosphere (e.g. mass extinctions).
For example, Brachiopoda was a dominant shelled filter feeding clade during the Paleozoic (514 to 252 million years ago; Mya) but was replaced by Bivalvia at the end Permian extinction event (252 Mya) which is now the dominant group (\citealt{Sepkiski1981,CLAPHAM01102006} but see \citealt{Payne22052014}).
This type of replacement pattern has also been observed in other groups such as Formaninifera \citep{Coxall01042006}, Ichtyosauria \citep{thorneresetting2011} or Plesiosauria \citealt{bensonfaunal2014} and are often related to competition \citep{brusatte50} or adaptive radiations \citep{Losos2010}.
Another classical example is the ``replacement'' of the dominant non-avian dinosaurs by mammals after the infamous Cretaceous-Paleogene (K-Pg) extinction 66 Mya...

\subsection{Cretaceous-Palaeogene extinction does not affect mammalian disparity}
In this fourth chapter, I studied the changes of morphological diversity \citep[or disparity;][]{Wills1994} through time using Total Evidence trees from \cite{slaterphylogenetic2013} and \cite{beckancient2014} to test whether the K-Pg extinction had an effect on mammal evolution.
I propose a new approach to describe patterns of disparity through time base on the use of Total Evidence trees.
This approach allows more precision in describing the changes through time as well as more freedom for choosing the underlining models of morphological evolution \citep[e.g. punctuated or gradual;][]{Hunt21042015}.
Using this approach I found no evidence of changes in disparity in mammals around the K-Pg boundary, arguing that the extinction of non-avian dinosaurs had no direct effect on mammalian evolution.\\

%\section{Discussion} % TG: I think if the title stays as shitty, no need for a title.
This is just one example of the benefits of adding both living and fossil taxa in macroevolutionary studies.
In the fifth chapter, I will discuss how the three previous chapters open new axis of research % TG: or does that sound super pretentious?
as well as the limitation of these studies.
Finally I will present some concluding thoughts on the utility of combining data, methods and disciplines to better understand macroevlutionary patterns and processes.

%\bibliography{References}