
\chapter{Data collection and processing}
\label{chap:methods}



\section{Overview}

	I compiled a morphological data set of both photographs and linear measurements from 366 specimens representing 99 species of small mammals. 
	%(total number of skulls and species in Access Data base, excluding the two _sp.)
	I collected morphological data from skulls, limbs and skins. I have divided my description of how the data were collected and processed into three sections:
	
	\begin{enumerate}[i.]
	
	\item Data collection (section \ref{sect:datacollection}): \\
	Summary of the species measured, taxonomic identification, photographic set up and image processing.
	
	\item Geometric morphometric analyses (section \ref{sect:morphometrics}):\\
	Landmark and semilandmark placement on different views of skulls and mandibles.
	
	
	\item Additional data collection (Appendix \ref{appendix}) \\
	In this thesis I only analysed a subset of the data I collected. Therefore, the additional data represent a significant resource for future work. I have documented collection of these data (linear measurements of skulls and limbs and photographs of skins) in Appendix \ref{appendix} at the end of my thesis.

	\end{enumerate} 


%####################################################
\section{Data collection}
\label{sect:datacollection}
%##################################################


\subsection{\normalfont{Species measured and taxonomy}}

	Between January and September 2013, I spent a total of nine weeks working in the collections of five museums: the Natural History Museum, London (BMNH), the Smithsonian Institute Natural History Museum, Washington D.C. (SI), the American Museum of Natural History, New York (AMNH), the Museum of Comparative Zoology, Cambridge M.A. (MCZ) and the Field Museum of Natural History, Chicago (FMNH). I measured and photographed 366 skulls, 248 post-cranial skeletons and 277 skins from 99 species belonging to seven Families within four Orders: Afrosoricida (Tenrecidae, Chrysochloridae), Erinaceomorpha (Erinaceidae), Soricomorpha (Soricidae, Solenodontidae, Talpidae) and Notoryctemorphia (Notoryctidae). 

	I measured all the tenrecs and golden moles available in the collections (31 of the 34 species of tenrec, and 12 of the 21 species of golden moles, Table \ref{tab:species.measured}). 
	For my comparative species of non-Afrosoricida species, I chose a random sample of 56 taxa which have been previously identified as convergent with tenrecs \citep[e.g.][]{Gould1966, Symonds2005, Poux2008, Olson2013}. I only analysed the Afrosoricida species for this thesis, details about the additional taxa can be found in Appendix \ref{appendix}.
	
	I recorded species names as they were written on museum specimen labels and then corrected them to match the taxonomy in Wilson and Reeder's Mammal Species of the World \citeyearpar{Wilson2005}. For recently identified species,  which are not included in Wilson and Reeder \citeyearpar{Wilson2005}, I used the taxonomy recorded on the specimen labels. Wilson and Reeder \citeyearpar{Wilson2005} record 30 species of tenrec but more recent studies indicate that there are now 34 species \citep[][Table \ref{tab:species.measured}]{Olson2013}. The additional species belong to the shrew tenrec (\textit{Microgale}) Genus and represent either recognition of cryptic species boundaries \citep{Olson2004} or discovery of new species \citep{Goodman2006, Olson2009}. Only one of these four recent additions, \textit{M. jobihely}, was present in the museum collections and therefore I could not include the three other newly recognised species in my analyses. Table \ref{tab:species.measured} outlines the number of species I measured from each Family and how this sample relates to the overall number of species in that Family.

%------------------------------------------------
%Species.measured Table

%I added the %coverage

\begin{table}[h]
	\caption[Species measured] 
	{The number of species measured in each Family compared to the total number of species in that Family. The total number of species in each Family is from \citet{Wilson2005} except for the Tenrecidae: there are now 34 species compared to 30 recognised in Wilson and Reeder (\citeyear{Wilson2005}).}
	\input{Methods/tables/species.measured}	
	\label{tab:species.measured}
\end{table}
 
%-----------------------------------------#

	%Moved this paragraph up from the error checking
	I included both male and female specimens in my data as I am interested in cranial shape of the species as a whole regardless of whether or not there are differences due to sexual size dimorphism. Age classification of mammal skulls is usually based on dental characteristics and cranial fusion but it is difficult to classify tenrecs in this way. In some species the last molar does not erupt fully until the first molar has been shed so the full dentition is never present at any one time \citep{Nowak1983}. Furthermore, it is difficult to distinguish deciduous from permanent teeth in \textit{Microgale} tenrecs \citep{Asher2008} which has led to confusion and misidentification of juvenile forms as separate species \citep{Olson2004}.
	I identified and excluded any obviously juvenile specimens based on incomplete cranial fusion. When specimens could not be obviously identified as juveniles I treated them all as equivalent adult forms. 	



%------------------------------------------------
%Moved the specimen labels and linear measurements into the appendix
%-----------------------------------------
\subsection{\normalfont{Photographing specimens}}

	To get 2D landmarks for my specimens, I first photographed them. I used photographic copy stands consisting of a camera attachment with an adjustable height bar, a flat stage on which to place the specimen and an adjustable light source. To take possible light variability into account, on each day I took a photograph of a white sheet of paper and used the custom white balance function on the camera to set the image as the baseline "white" measurement for those particular light conditions.
	
	I photographed the specimens with a Canon EOS 650D camera fitted with a EF 100 mm f/2.8 Macro USM lens. I used a remote control (H\"ahnel Combi TF) to take the photos to avoid shaking the camera and distorting the images. I photographed the specimens on a black material background. I placed the light source from the top left-hand corner of the photograph and positioned a piece of white card on the bottom right side of the specimen which reflected the light back onto the specimen and minimised any shadows (Figure \ref{fig:camera}). I used small bean bags as necessary to hold the specimens in position while being photographed to ensure that they lay in a flat plane relative to the camera and did not tilt in any direction. I used the grid-line function on the live-view display screen of the camera to position the specimens in the centre of each image. 

%----------------------------------------------
%Camera picture % NC: Can probably make this picture smaller - currently it's half a page.
%SF I reduced it from 12 to 10 cm
\begin{figure}[h] 
  \centering
  \includegraphics[width=10cm, height=10cm, keepaspectratio=true]{Methods/figures/camera.jpg}
    \caption[Photographic set up]
    {Photographic set up for taking pictures of skulls. The camera (above centre) is fitted to a copy stand, the light source is directed from the top-left corner of the image and the white card reflects the light back onto the skull. }
  \label{fig:camera}
  \end{figure}
%-------------------------------------------------

	I photographed the skulls in three views: dorsal (top of the cranium), ventral (underside of the skull with the palate roof facing upwards) and lateral (right side of the skull). I also photographed the outer (buccal) side of the right mandible. When the right sides of either the skulls or mandibles were damaged or incomplete I photographed the left sides and later reflected the images so that they could be compared to pictures of the right sides \citep[e.g.][]{Barrow2008}.


\subsection{\normalfont{Saving and processing images}}
	Photographs were captured and saved in raw file format. Before using the photos for morphometric analyses, I converted the raw files to binary (grey scale) images and re-saved them as TIFF files. The black and white images were more useful for later analyses since I was not interested in including any colour comparisons and it is easier to see some biological features in binary images. TIFF files were also appropriate because they are uncompressed (in comparison to JPEG) images and therefore there is less chance of any picture distortions which may affect later analyses \citep{RHOI2013}.
	Photographs of the specimens from the American Museum of Natural History and the Smithsonian Institute are available on figshare in separate file sets for the dorsal \citep{Finlay2013d}, ventral \citep{Finlay2013v} and lateral \citep{Finlay2013l} skull pictures along with the mandibles \citep{Finlay2013m}. Copyright restrictions from the other museums prevent public sharing of their images however they are available on request.
	

%##################################################

\section{Geometric morphometric analyses}
\label{sect:morphometrics}

\subsection{\normalfont{Landmark placement on images}}


	I used a combination of landmark and semilandmark analysis approaches to assess the shape variability in my skull and mandible specimens.  I used the thin plate spline (TPS) software suite \citep{Rohlf2013} to digitise landmarks and curves on the photos. I set the scale on each image individually to standardise for the different camera heights that I used when photographing my specimens. I created separate data files for each of my four morphometric analyses (skulls in dorsal, ventral and lateral views and mandibles in lateral view). I digitised landmarks (discrete points) and semilandmarks \citep[points on a geometric feature, such as a curve, which are defined by their relative position on that feature,][]{Zelditch2012} on every image individually. I selected my landmarks based on their relevance to the overall anatomical shape of the specimen and the ease with which they could be reliably identified on different species. Some specimens were too damaged to use in particular views so I had a different total number of images for each analysis: skulls dorsal (356), ventral (346), lateral (336) and mandibles (356). 
	%Numbers are from the two_family_disparity script after cleaning up the data to remove damages specimens

	When using semilandmark approaches there is a potential problem of over -\\sampling: simpler structures will require fewer semilandmarks to accurately represent their shape  \citep{MacLeod2012}. To ensure that I applied a uniform standard of shape representation to each outline segment (i.e. that simple structures would not be over-represented and more complex features would not be under-represented), I followed the method outlined by MacLeod \citeyearpar{MacLeod2012}. For each data set I chose a random selection of photos of specimens which represented the breadth of the morphological data (i.e. specimens from each sub-group of species). I drew the appropriate curves on each specimen and over-sampled the number of points on the curves. I measured the length of the line and regarded that as the 100\%, true length of that outline. I then re-sampled the curves with decreasing numbers of points and measured the length of the outlines. I calculated the length of each re-sampled curve as a percentage of the total length of the outline and then found the average percentage length for that reduced number of semilandmark points across all of the specimens in my test file. I continued this process until I found the minimum number of points that gave a curve length which was at least 95\% accurate. This 95\% cut off acts as a balance between accurate representation of the curve and avoiding unnecessary over-sampling \citep{MacLeod2012}. I repeated these curve-sampling tests for each analysis to determine the minimum number of semilandmark points which would give accurate representations of morphological shape.
	
	Here I summarise the landmarks and curves which I used on each of my different sets of photos. For landmarks which are defined by dental structures, I used published dental sources \citep{Repenning1967, Eisenberg1969, Nowak1983, MacPhee1987, KnoxJones1992, Davis1997, Querouil2001, Nagorsen2002, Wilson2005, Goodman2006, Karatas2007, Hoffmann2008, Asher2008,  Muldoon2009, Lin2010} where available to identify the number and type of teeth in each species.
	
%------------------------------------------------------------------
\subsection{\normalfont{Skulls: dorsal view}}
	Most of my landmarks in this view were relative (type 3) points which represent overall morphological shape but not necessarily homologous biological features \citep{Zelditch2012}. I placed ten landmarks and drew four semilandmark curves to represent the shape of both the braincase (posterior) and nasal (anterior) area of the skulls (Figure \ref{fig:skdors_skvent}). Table \ref{tab:skdors} describes how I placed the landmarks and drew the outline curves for my images of skulls in dorsal view.

\subsection{\normalfont{Skulls: ventral view}}

	Most of the landmarks in this view are concentrated around the dentition and palate of the animals. I placed 13 landmarks and drew one outline curve (resampled to 60 semilandmark points) around the back of the skull between landmarks 12 and 13 (Figure \ref{fig:skdors_skvent}). The high variability of my species' basi-cranial regions and difficulties associated with identifying developmentally or functionally homologous points precluded designation of additional landmarks towards the back of the skulls. Table \ref{tab:skvent} outlines the descriptions of the landmarks I placed on the ventral skull photos.

%------------------------------------------------
%Skulls dorsal and ventral

\begin{figure}[!htbp]
	\centering
	\includegraphics[width=1\linewidth]{Methods/figures/Skdors+Skvent_combined_BW.png}
	\caption[Skulls: dorsal and ventral landmarks]
	{Landmarks (numbered points) and curves (outlines) for the skulls in dorsal and ventral view. See Tables \ref{tab:skdors} and \ref{tab:skvent} for landmark descriptions. The skulls are two different specimens of \textit{Potamogale velox} (otter shrew tenrec), museum accession numbers AMNH 51327 and BMNH 1934.6.16.2. }
	\label{fig:skdors_skvent}
\end{figure}


%Skdors descriptions
\begin{table}[h]
	\caption[Skulls: dorsal landmarks]
		{Descriptions of the landmarks (points) and curves (semilandmarks) for the skulls in dorsal view (Figure
		\ref{fig:skdors_skvent})} 
	\input{Methods/tables/skdors.landmarks}
	\label{tab:skdors}
\end{table}

%Skvent descriptions
% NC: Fixed your footnote here
\begin{table}[!htb] %force the table to go up with the picture
\caption[Skulls: ventral landmarks]
		{Descriptions of the landmarks (points) and curves (semilandmarks) for the skulls in ventral view (Figure \ref{fig:skdors_skvent}).} 
\input{Methods/tables/skvent.landmarks}
\label{tab:skvent}
\\
*This curve does not necessarily trace homologous features because of the variation in the position of the foramen magnum.
\end{table}
%-------------------------------------------------------------

\subsection{\normalfont{Skulls: lateral view}}
	I placed nine landmarks on the photographs of skulls in lateral view (Figure \ref{fig:sklat_mands}) and also drew two semilandmark curves to represent the shape of the back of the skull (between landmarks 7 and 8, re-sampled to 20 semilandmark points) and the top of the skull (between landmarks 8 and 1, re-sampled to 15 semilandmark points). Table \ref{tab:sklat} describes my definitions for each of the landmark points.
	If specimens were damaged on their right side I reflected photographs of the left lateral side of the skull so that all the images were in the same orientation.


\subsection{\normalfont{Mandibles}}
	I placed seven landmarks and drew four curves on each mandible photograph (again, reflecting any images of the left mandible so they were in the correct orientation, Figure \ref{fig:sklat_mands}). I drew separate curves around each of the three processes of the ascending ramus: coronoid, condyloid and angular and along the base of the horizontal ramus of the jaw (Figure \ref{fig:sklat_mands}, Table \ref{tab:mands}). 	
%SF: Took out the paragraph about developmental independence in the jaw

%---------------------------------------------------
%Combined sklat and mandibles diagrams
\begin{figure}[!htbp]
	\centering
	\includegraphics[width=1\linewidth]{Methods/figures/Sklat+mands_combined_BW.png}
	\caption[Lateral skulls and mandibles landmarks]
	{Landmarks (numbered points) and curves (outlines) for the skulls and mandibles in lateral view. See tables \ref{tab:sklat} and \ref{tab:mands} for landmark descriptions. The skull (BMNH 1934.6.16.2.) and mandible (AMNH 51327) belong to two different specimens of \textit{Potamogale velox} (otter shrew tenrec).}
	\label{fig:sklat_mands}
\end{figure}

%Sklat descriptions
\begin{table}[!htb]
\caption[Skulls: lateral landmarks]
		{Descriptions of the landmarks (points) and curves (semilandmarks) for the skulls in lateral view (Figure \ref{fig:sklat_mands}).} 
\input{Methods/tables/sklat.landmarks}
\label{tab:sklat}
\end{table}

%Mandibles descriptions
\begin{table}[!htb]			
	\centering
	\caption[Mandibles: landmarks]
		{Descriptions of the landmarks (points) and curves (semilandmarks) for the mandibles in lateral (buccal) view (Figure \ref{fig:sklat_mands}).}
	\input{Methods/tables/mandibles.landmarks}
	\label{tab:mands} 
\end{table}
%------------------------------------


\newpage
\subsection{\normalfont{Procrustes superimposition}}
\label{sect:procrustes}

	After creating my files with the landmarks and semilandmarks placed on each photograph, I used the thin plate spline utilities software (TPSUtil) \citep{Rohlf2012} to create "sliders" files that defined which points in the TPS files should be treated as semilandmarks \citep{Zelditch2012}. I conducted all further morphometric analyses in R version 3.1.1 \citep{Team2014} within the geomorph package \citep{Adams2013}.

	
	I used the \texttt{gpagen} function in the geomorph package \citep{Adams2013} to run a general Procrustes alignment \citep{Rohlf1993} of the landmark coordinates while sliding the semilandmarks by minimising Procrustes distance \citep{Bookstein1997}. Procrustes superimposition creates a common set of Procrustes-superimposed landmarks that can be used to compare differences in shape that are independent of differences in size, location and orientation of the specimens \citep{Webster2010}. It is necessary to slide semilandmarks because equidistant points along a curve do not necessarily correspond to geometric or biological homology across specimens \citep{Gunz2013}. Sliding semilandmarks along a curve during Procrustes superimposition removes this effect of arbitrary semilandmark spacing by optimising the position of the semilandmarks with respect to the average shape of the entire sample (the average of all the Procrustes shape coordinates). Sliding semilandmarks by minimising Procrustes distance \citep[the sum of squared distances between corresponding points of two Procrustes-superimposed shapes,][]{Zelditch2012} allows each semilandmark to slide separately without being influenced by the position of other landmarks or semilandmarks \citep{Gunz2013}. 
	
	I used these Procrustes-aligned coordinates of all specimens to calculate average shape values for each species which I then used for a principal components (PC) analysis with the \texttt{plotTangentSpace} function \citep{Adams2013}. 
	

%------------------------------------------------
\subsection{\normalfont{Potential morphometrics errors}}
	 
	While 2D methods are an accepted means of comparing morphological shape \citep[e.g.][]{Adams2004, Mitteroecker2009}, particularly for comparing skull morphologies of small mammals \citep[e.g.][]{Cardini2003, Panchetti2008, White2008, Barrow2008, Scalici2011}, the inherent discrepancies associated with comparing three dimensional objects using two dimensional pictures can introduce some potential problems of possible image distortion \citep{Arnqvist1998}. Similarly, human error with how landmarks are positioned on specimens could also introduce noise into further analyses. 
	
	In contrast to detailed intraspecific work \citep[e.g.][]{Bornholdt2008, Blagojevic2011}, photographic or landmark placement errors are unlikely to be significant in my interspecific study since one would expect that the morphological variation among species is large enough to  be detected as a signal above any background noise associated with methodological error \citep{Arnqvist1998}. Nevertheless, it is still important to assess measurement error in a morphometric data set to increase confidence in the outcome of final analyses.
	I identified two possible sources of morphometric measurement error: specimen orientation and placement of landmarks.

	Variation in the orientation of specimens for photography is one of the main sources of error in 2D morphometric studies \citep{Adriaens2007}. If specimens are not placed on a flat plane or in a consistent position relative to the camera, areas of the object which are tilted towards the camera will appear to be larger than in reality, distorting any subsequent morphometric analyses of the shape. 
	I placed the landmarks on each set of pictures so inter-observer variation in landmark placement is not an issue for my study.  However, repeatability and reliability of my choice of landmarks could affect the final results of my analyses \citep{Arnqvist1998}.


	To measure potential orientation error, I photographed the skulls (dorsal, ventral and lateral views) and mandibles of each specimen three times, cycling through the photos so that the specimen was removed and re-positioned before every shot \citep{Viscosi2011}.
	I used a subset of my ventral skull images to test for two sources of error: specimen orientation and landmark placement error \citep{Arnqvist1998, Barrow2008}. Of the specimens which I photographed three times,  I chose a random subset of seven skulls from four different Families: three tenrecs and single representatives of shrews, moles, hedgehogs and golden moles. I copied these images and placed landmarks on three copies of each image to compare variation in landmark placement within each orientation. This gave nine replicates of each skull: three separate photos, and three copies of each of those photos. As before, I ran a general Procrustes alignment \citep{Rohlf1993} of the specimens, calculated the average shape values for each skull (average of all nine pictures) and used these values for a principal components (PC) analysis. 
		
	I used a linear mixed effects model to model the shape variation (first PC axis) associated with specimen (fixed effect) and two nested random effects: photo identity nested within specimen and image replicate nested within each photograph. This nested approach was necessary to deal with the hierarchical nature of my data subset: three photos of every specimen and three replicates of each of those images. I ran the test using the lmer function in the lme4 package \citep{Bates2014}.  

	I found that specimen orientation and landmark placement have negligible effects on the overall shape variation between different skulls: the random effect variables of photo identity and image replicate explained less than 0.0001 \% of the overall shape variation among different skulls. Therefore, I am confident that shape variation among the specimens in the rest of my analyses reflects true morphological differences rather than methodological error.







